packages/xo-web/src/common/modal.js

```diff
@@ -368,7 +368,9 @@ export const FormModal = decorate([
   injectState,
   ({ state, effects }) => (
     <ReactModal
+      backdrop='static'
       bsSize={state.size}
+      keyboard={false}
```

Comment:
Why `false`? The `keyboard` closes the modal when escape key is pressed.

Reply:
You can set only `backdrop` to `static` to resolve it, no?

Reply:
- `backdrop='static'` keeps the modal open when you click outside
- `keyboard={false}` keeps the modal open when you press escape. I added this one because I think it's also a "I accidentally closed the modal" case. Open to discussion, though.

Reply:
According to [this doc](https://devdocs.io/bootstrap~4/components/modal/index#backdrop-option), `backdrop='static'` is enough.
But, it's not bad to keep it.

Reply:
That's the documentation for Bootstrap but we use react-boostrap which [has a slightly different behaviour](https://react-bootstrap-v3.netlify.app/components/modal/#modals-props-modal).